generators/generator-constants.js

```diff
@@ -49,36 +49,62 @@ const ELASTICSEARCH_VERSION = '7.15.2'; // The version should be coherent with t
+const DOCKER_HAZELCAST_MANAGEMENT_CENTER_VERSION = '4.2022.01';
+const DOCKER_HAZELCAST_MANAGEMENT_CENTER = `hazelcast/management-center:${DOCKER_HAZELCAST_MANAGEMENT_CENTER_VERSION}`;
+const DOCKER_MEMCACHED_VERSION = '1.6.15-alpine';
+const DOCKER_MEMCACHED = `memcached:${DOCKER_MEMCACHED_VERSION}`;
+const DOCKER_REDIS_VERSION = '6.2.6';
+const DOCKER_REDIS = `redis:${DOCKER_REDIS_VERSION}`;
+const DOCKER_KEYCLOAK_VERSION = '18.0.0';
+const DOCKER_KEYCLOAK = `quay.io/keycloak/keycloak:${DOCKER_KEYCLOAK_VERSION}`;
 const DOCKER_ELASTICSEARCH_CONTAINER = 'docker.elastic.co/elasticsearch/elasticsearch';
 const DOCKER_ELASTICSEARCH = `${DOCKER_ELASTICSEARCH_CONTAINER}:${ELASTICSEARCH_VERSION}`;
 const DOCKER_KAFKA = `confluentinc/cp-kafka:${KAFKA_VERSION}`;
 const DOCKER_ZOOKEEPER = `confluentinc/cp-zookeeper:${KAFKA_VERSION}`;
-const DOCKER_SONAR = 'sonarqube:9.4.0-community';
-const DOCKER_CONSUL = 'consul:1.12.0';
-const DOCKER_CONSUL_CONFIG_LOADER = 'jhipster/consul-config-loader:v0.4.1';
-const DOCKER_PROMETHEUS = 'prom/prometheus:v2.35.0';
-const DOCKER_PROMETHEUS_ALERTMANAGER = 'prom/alertmanager:v0.24.0';
-const DOCKER_GRAFANA = 'grafana/grafana:8.5.0';
-const DOCKER_JENKINS = 'jenkins/jenkins:lts-jdk11';
-const DOCKER_SWAGGER_EDITOR = 'swaggerapi/swagger-editor:latest';
-const DOCKER_PROMETHEUS_OPERATOR = 'quay.io/coreos/prometheus-operator:v0.42.1';
-const DOCKER_GRAFANA_WATCHER = 'quay.io/coreos/grafana-watcher:v0.0.8';
-const DOCKER_ZIPKIN = 'openzipkin/zipkin:2.23';
+const DOCKER_SONAR_VERSION = '9.4.0-community';
+const DOCKER_SONAR = `sonarqube:${DOCKER_SONAR_VERSION}`;
+const DOCKER_CONSUL_VERSION = '1.12.0';
+const DOCKER_CONSUL = `consul:${DOCKER_CONSUL_VERSION}`;
+const DOCKER_CONSUL_CONFIG_LOADER_VERSION = 'v0.4.1';
+const DOCKER_CONSUL_CONFIG_LOADER = `jhipster/consul-config-loader:${DOCKER_CONSUL_CONFIG_LOADER_VERSION}`;
+const DOCKER_PROMETHEUS_VERSION = 'v2.35.0';
+const DOCKER_PROMETHEUS = `prom/prometheus:${DOCKER_PROMETHEUS_VERSION}`;
+const DOCKER_PROMETHEUS_ALERTMANAGER_VERSION = 'v0.24.0';
+const DOCKER_PROMETHEUS_ALERTMANAGER = `prom/alertmanager:${DOCKER_PROMETHEUS_ALERTMANAGER_VERSION}`;
+const DOCKER_GRAFANA_VERSION = '8.5.0';
+const DOCKER_GRAFANA = `grafana/grafana:${DOCKER_GRAFANA_VERSION}`;
+const DOCKER_JENKINS_VERSION = 'lts-jdk11';
+const DOCKER_JENKINS = `jenkins/jenkins:${DOCKER_JENKINS_VERSION}`;
+const DOCKER_SWAGGER_EDITOR_VERSION = 'latest';
+const DOCKER_SWAGGER_EDITOR = `swaggerapi/swagger-editor:${DOCKER_SWAGGER_EDITOR_VERSION}`;
+const DOCKER_PROMETHEUS_OPERATOR_VERSION = 'v0.42.1';
+const DOCKER_PROMETHEUS_OPERATOR = `quay.io/coreos/prometheus-operator:${DOCKER_PROMETHEUS_OPERATOR_VERSION}`;
+const DOCKER_GRAFANA_WATCHER_VERSION = 'v0.0.8';
+const DOCKER_GRAFANA_WATCHER = `quay.io/coreos/grafana-watcher:${DOCKER_GRAFANA_WATCHER_VERSION}`;
+const DOCKER_ZIPKIN_VERSION = '2.23';
+const DOCKER_ZIPKIN = `openzipkin/zipkin:${DOCKER_ZIPKIN_VERSION}`;
 
```

Comment:
generator-constants is too bloated we should move to `docker-images.js` and consider a dependabot auto update for v8.

For v7 compatibility just re-export
```
module.exports = {
  ...require('./docker-images.js'),
};
```

Reply:
Why not, but in another PR